### ISSUE TITLE
association scheme variables Na, Nb, Nc

### ISSUE THREAD

**Issue by iurisegtovich** [feos-org/feos#220]
hi, i have a question regarding association scheme variables Na, Nb, Nc:

from "rehner2020.json", water _2B, _3B and _4C, I understand that NA is the number of electron accepting sites (~H) and NB is the number of electron donating sites ( ~O: ), but what is the NC field (currently set to default zero)?

![rehner2020_water4c](https://github.com/feos-org/feos/assets/10930517/50b67c8a-e5ab-4700-a595-8c0b3a14b4ea)

as I understand, all interactions are calculated from a single couple of parameters for each molecule (kappa_ab and epsilon_k_ab) and these sites numbers and its intrinsic characteristics.

thanks,

iuri.

**Comment by g-bauer** [comment]
Hi Iuri,

@prehner created a nice representation [here](https://feos-org.github.io/feos/theory/models/association.html). Does that help?

**Comment by iurisegtovich** [comment]
yes, that material is exactly what i was looking for, thanks! :)

**Comment by g-bauer** [comment]
Great, I'm closing this issue. We can open it again if new questions come up.

**Comment by iurisegtovich** [comment]
Hi, I have recently been comparing schemes for mixtures,
and in the linked presentation it is stated that "The CC association is less widely used but implemented to ensure that all the association schemes defined in Huang and Radosz 1990 are covered."
but in that paper only self association is discussed between molecules with A and B or with only C sites,
therefore mixtures are up to intepretation.

Mixtures are presented in Huang and Radosz 1991 but without cross association between acids and water.
> "...In this work, we do not include systems containing cross associations, such as acids + water. These systems will be addressed later in the continuation of this work."

Then, according to the interpretation of later papers (Wolbach, and  Sandler. 1998) it can be extended to mixtures by considering that C interacts with A and B as well (for example one can call A as (+), B as (-), and C as (+-), so +- interacts with all of (+), (-), (+-)
```
Wolbach, Jeffrey P., and Stanley I. Sandler.
"Using molecular orbital calculations to describe the phase behavior of cross-associating mixtures."
Industrial & engineering chemistry research 37.8 (1998): 2917-2928.
```
https://doi.org/10.1021/ie970781l -> table 4 -> assumptions about cross interactions between molecules of type 1 (acids) with any other.

therefore I believe one would need the following scheme to reproduce Wolbach & Sandler work:

|   | A  | B  | C  |
|---|----|----|----|
| **A** |    | Δᴬᴮ  |  Δᴬᶜ  |
| **B** | Δᴮᴬ |    | Δᴮᶜ   |
| **C** |  Δᶜᴬ  | Δᶜᴮ   | Δᶜᶜ |

with the usual symmetry /  matrix transposition  rules

what do you guys think about that?

are there any validation cases that rely on the Δᶜᴬ  = Δᶜᴮ = assumption ?

**Comment by prehner** [comment]
Hi Iuri, fair point. The comment in the documentation only refers to pure components. I'm traveling right now without a laptop, so I didn't completely look through this, but if we assume the c association site to be +-, couldn't we obtain the identical set of equations by putting A and B sites and assume we have full control over the binary association parameters for every site/site combination?

**Comment by iurisegtovich** [comment]
Thanks for acknowledging the point and re-opening the issue.
 I'll be running some test calculations with my colleagues, including those involving acids, and will share further thoughts after that.
